src/Soil/Soil.jl

```diff
@@ -156,12 +156,15 @@ For column domains, no dss is needed.
 function dss!(dY::ClimaCore.Fields.FieldVector, domain::Column) end
 
 """
-   dss!(dY::ClimaCore.Fields.FieldVector,domain::HybridBox)
+   dss!(dY::ClimaCore.Fields.FieldVector,domain::{HybridBox, SphericalShell})
```

Comment:
Missing a `Union`

Reply:
We could subtype `AbstractDomain` with something like `AbstractHybridDomain`, perhaps? Depends on what the common feature of these domains and their functionality is.

Reply:
Yes, this could be useful! Especially when we rearrange rhs operations to loop over columns, it might come up more often. The common feature is that they are extrudedFD spaces, with a spectral horizontal space, like you said. 

Maybe one other thing to check - what does dss! do if we are on a column? Would it be confusing if we do it all the time, if it doesnt do anything for a column field?

Reply:
Right, we have it written that `dss!` does nothing on the column but actually does dss for the other domains. I don't think this is particularly confusing and the great benefit is that we have a single RHS.

Reply:
ok cool! once we do the "loop over columns" PR we can simplify this too, or before then if it makes sense